pytorch_lightning/trainer/connectors/accelerator_connector.py

```diff
@@ -413,6 +413,17 @@ def _check_device_config_and_set_final_flags(
         self._num_nodes_flag = int(num_nodes) if num_nodes is not None else 1
         self._devices_flag = devices
 
+        if self._devices_flag in ([], 0, "0"):
+            accelerator_name = (
+                self._accelerator_flag.__class__.__qualname__
```

Comment:
just wondering, is there a difference between `type(self._accelerator_flag)` and `self._accelerator_flag.__class__`?

From my understanding they should return the same. If they do, personally I have a slight preference towards `type(...)` just because it doesn't involve calling magic attributes manually :)

Reply:
it's exactly the same. the attribute access is probably faster but this is all cherry-picking. Up to you

Reply:
Then it's totally fine :) I've just been curious, thanks @carmocca